EIPS/eip-3155.md

```diff
@@ -14,7 +14,9 @@ created: 2020-12-07
 Introduce a new JSON standard for EVM traces during execution of state tests. 
 
 ## Motivation
-The Ethereum Virtual Machine executes all smart contract code on ethereum. In order to debug smart contracts and state tests better, a common format was introduced to log every execution step of the EVM. This format was implemented by go-ethereum, parity and nethermind and Besu. Since the common format was not well defined, the implementations differed slightly making it hard to develop adequate tooling.
+The Ethereum Virtual Machine executes all smart contract code on ethereum. In order to debug smart contracts and state tests better, a common format was introduced to log every execution step of the EVM. This format was implemented by go-ethereum, parit, nethermind and Besu. Since the common format was not well defined, the implementations differed slightly making it hard to develop adequate tooling.
+This reduces the usefulness of tracing significantly.
```

Comment:
```suggestion
The Ethereum Virtual Machine executes all smart contract code on ethereum. In order to debug smart contracts and state tests better, a common format was introduced to log every execution step of the EVM. This format was implemented by go-ethereum, parit, nethermind and Besu. Since the common format was not well defined, the implementations differed slightly making it hard to develop adequate tooling, which reduces the usefulness of tracing significantly.
```